rollbar/__init__.py

```diff
@@ -1133,6 +1166,11 @@ def _send_payload(payload, access_token):
         _post_api('item/', payload, access_token=access_token)
     except Exception as e:
         log.exception('Exception while posting item %r', e)
+    try:
```

Comment:
If we are using threads this will be called on one of the threads created for sending a payload. `get_nowait` will remove a thread from the queue, it doesn't really matter which one it removes as long as each thread calls this exactly once because we don't do anything with the returned value. Again, since we are on one of the threads, calling task_done here can signal to join to return.

If we are not using threads, i.e. any other handler, then _threads will just always be empty and get_nowait will throw queue.Empty and we just move on.